crates/build/re_types_builder/src/data_type.rs

```diff
@@ -153,8 +153,10 @@ impl DataType {
 pub enum LazyDatatype {
     Atomic(AtomicDataType),
 
+    /// A list of bytes of arbitrary length.
```

Comment:
Just for visibility (you don't have to change anything): The encoding is not just a list of bytes, but a [prefixed list of bytes](https://arrow.apache.org/docs/cpp/api/datatype.html):

> Bytes view type with 4-byte prefix and inline small string optimization.

Might only ever be important in cases were we want to memory map something.

Reply:
You're reading the docs for "BinaryView", which is something else